scripts/go-function-release.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+scripts_dir="$(dirname "$0")"
+# git-tag-parser.sh has been shell-checked separately.
+# shellcheck source=/dev/null
+source ${scripts_dir}/git-tag-parser.sh
+
+DEV_TAG=dev
+versions=$(get_versions "${TAG}")
```

Comment:
wheres's TAG coming from? Each bash script should have a top-level description describing its usage including any  env vars it expects.

Reply:
> wheres's TAG coming from?

In `function-release.sh`, we set TAG. `CURRENT_FUNCTION="${fn_name}" TAG="${fn_ver}" make func-build`.